docs/event-notification.md

```diff
@@ -0,0 +1,26 @@
+# Event notification
+
+REMS can be configured to send events notifications over HTTP.
```

Comment:
Hmm, with the current outbox model we can't really guarantee ordering. I'll add notes for the guarantees we do have. Guaranteed ordering (per-endpoint) might be useful, but it would also possibly cause weird delays (we'd retry a failing event up to 48h before sending the next ones). Let's talk about this a bit more.

Reply:
Updated the docs to match the implementation.